plugins/win-capture/game-capture.c

```diff
@@ -185,6 +200,10 @@ struct game_capture {
 struct graphics_offsets offsets32 = {0};
 struct graphics_offsets offsets64 = {0};
 
+static void send_game_capture_status_event(struct game_capture *gc);
+static void waiting_image_unload(struct game_capture *gc);
+static void waiting_image_load(struct game_capture *gc);
```

Comment:
Having troubles to understand why these 2 functions are being prefixed by `waiting` when there is no actual wait inside.

Reply:
names updated.